DSCResources/MSFT_xSQLAOGroupEnsure/MSFT_xSQLAOGroupEnsure.psm1

```diff
@@ -0,0 +1,351 @@
+$currentPath = Split-Path -Parent $MyInvocation.MyCommand.Path
```

Comment:
This change enables the ability to deploy/configure AlwaysOn availability groups